test/tests/functional/pbs_qsub_direct_write.py

```diff
@@ -64,6 +64,126 @@ def test_direct_write_when_job_succeeds(self):
         file_count = len([name for name in os.listdir(
             mapping_dir) if os.path.isfile(os.path.join(mapping_dir, name))])
         self.assertEqual(2, file_count)
+        self.server.expect(JOB, {'Keep_Files': 'doe'}, id=jid)
+
+    def test_direct_write_output_file(self):
+        """
+        submit a sleep job and make sure that the output file
+        are getting directly written directly to the mapped
+        directory when direct_files option is used with o option.
+        """
+        j = Job(TEST_USER)
+        j.set_attributes({ATTR_k: 'do'})
```

Comment:
Please use `j = Job(TEST_USER, attrs={ATTR_k: 'do'})` to combine the above two lines into one.